backend/socket.io/index.js

```diff
@@ -21,6 +21,9 @@ const initSocket = (server) => {
 const socketEmit = (topic, sid, data) => {
   const socket = io.sockets.sockets[sid];
   socket.emit(topic, data);
+  return socket.on('received', () => {
+    return true;
+  });
```

Comment:
Didn't understand what's the role of it or why it's required.

Reply:
Receiver emits 'received' after receiving chat-message

Reply:
`socketEmit` is a generic function let's not put it here, also `.on` is to listen to specific events.